src/components/DeliveryAddressDialog/DeliveryAddressDialog.jsx

```diff
@@ -110,22 +113,21 @@ const DeliveryAddressDialog = ({ onSelect }) => {
     try {
       const url = `${apiUrl}/api/users/addNewAddress`;
       const result = await axios.post(url, data);
-      console.log('resultvsdvsdvsd', result);
       if (result.status === 201) {
```

Comment:
Instead of session, we need to loggedInUser.address ? from the store ? If I'm wrong correct me  or we need to complete remove whole code and add mapbox?

Reply:
we will remove the whole code and add mapbox